pkg/client/clientset.go

```diff
@@ -156,8 +156,8 @@ type BucketClient interface {
 type DocumentClient interface {
 	Get(ctx context.Context, id string) (documents.Response, error)
 	List(ctx context.Context, filter string) (documents.ListResponse, error)
-	Create(ctx context.Context, name string, externalId string, data []byte, documentType documents.DocumentType) (documents.Response, error)
-	Update(ctx context.Context, id string, name string, data []byte, documentType documents.DocumentType) (documents.Response, error)
+	Create(ctx context.Context, name string, isPrivate bool, externalId string, data []byte, documentType documents.DocumentType) (documents.Response, error)
```

Comment:
I amplify my previous complaint about the increasing number of arguments.

Here are some thoughts:
 * https://matheus.ro/2018/01/29/clean-code-avoid-many-arguments-functions/
 * https://softwareengineering.stackexchange.com/a/145066/302528
 * https://dev.to/rweisleder/the-flag-parameter-anti-pattern-1j82